services/common/workflow.js

```diff
@@ -26,16 +26,16 @@ export const ONEMAC_LABEL = {
 
 export const ONEMAC_STATUS = {
   INACTIVATED: "Inactivated",
-  UNSUBMITTED: "Unsubmitted",
   SUBMITTED: "Submitted",
-  RAI_SUBMITTED: "RAI Submitted",
-  IN_REVIEW: "Under Review",
-  RAI_ISSUED: "RAI Issued",
+  PENDING: "Pending",
+  PENDING_CONCURRANCE: "Pending - Concurrance",
+  PENDING_APPROVAL: "Pending - Approval",
+  RAI_ISSUED: "Pending - RAI",
   APPROVED: "Approved",
   DISAPPROVED: "Disapproved",
-  WITHDRAWN: "Package Withdrawn",
```

Comment:
I suppose this doesnt matter since we are doing the mapping by user to completely different things and never showing these values, but just wondering why we dropped the Package off Package Withdrawn here?

Reply:
So... now that we have four sets of statuses and the true OneMAC status is only for viewing in the database (and for the mappings) ... I kinda felt being closer to the SEATool statuses would be more useful?